addons/test_mail/tests/test_performance.py

```diff
@@ -512,15 +512,15 @@ def test_complex_tracking_subscription_subtype(self):
             rec.write({
                 'name': 'Test2',
                 'umbrella_id': self.umbrella.id,
-                })
+            })
 
         self.assertEqual(rec.message_partner_ids, self.partners | self.env.user.partner_id | self.user_portal.partner_id)
         # write tracking message
         self.assertEqual(rec.message_ids[0].subtype_id, self.env.ref('test_mail.st_mail_test_full_umbrella_upd'))
         self.assertEqual(rec.message_ids[0].notified_partner_ids, self.partners | self.user_portal.partner_id)
         # creation message
-        self.assertEqual(rec.message_ids[1].subtype_id, self.env.ref('mail.mt_note'))
-        self.assertEqual(rec.message_ids[1].notified_partner_ids, self.env['res.partner'])
+        self.assertEqual(rec.message_ids[1].subtype_id, self.env.ref('test_mail.st_mail_test_full_umbrella_upd'))
```

Comment:
I would like 0 changes in test (only maybe query count) ...

Reply:
Agree, 0 change in test is better, but see answer above for this specific case.

Reply:
Resolved